package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ecosia/eslint-config",
-  "version": "3.0.0",
+  "version": "4.0.0",
```

Comment:
In my opinion this does not deserve being a major upgrade. I think `3.1` would be enough since it is not breaking anything. What do you think?

Reply:
agreed! I wasn't sure how we want to version this package - in a way, this is a breaking change as it changes the external behaviour of the package, but I agree it feels weird to have a major bump.

Reply:
I rebased and bumped to 3.1.0